### PR TITLE
[ME-166] Setup Jest and Enzyme for frontend unit testing

### PR DESCRIPTION
**Pull request by kulczewicz**

### CODE REVIEW

File: app/enzyme.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const Adapter = require("enzyme-adapter-react-16");
+
+require("enzyme").configure({ adapter: new Adapter() });
```
Comment (mecies) [inline app/enzyme.js:3]: Is es5 required here? How about: 
```javascript
import Adapter from "enzyme-adapter-react-16" 
import Enzyme from "enzyme" 

Enzyme.configure({ adapter: new Adapter() });
```
Also any specific reason for `.js` and not `.ts` file? 😄

File: app/package.json [hunk 2]
```diff
@@ -39,15 +41,21 @@
...
+    "@types/enzyme-adapter-react-16": "1.0.6",
+    "@types/jest": "25.2.3",
     "@types/js-cookie": "2.2.6",
     "@types/node": "13.13.2",
     "@types/react": "16.9.34",
     "@types/react-onclickoutside": "6.7.3",
     "babel-plugin-emotion": "10.0.33",
     "babel-plugin-jsx-remove-data-test-id": "2.1.3",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
```
Comment (mecies) [inline app/package.json:54]: For now, we stick to fixed versions, please remove `^`

File: app/package.json [hunk 3]
```diff
@@ -39,15 +41,21 @@
     "@babel/core": "7.9.0",
     "@project/eslint": "1.0.0",
     "@project/jest": "1.0.0",
+    "@types/enzyme": "^3.10.5",
```
Comment (mecies) [inline app/package.json:44]: For now, we stick to fixed versions, please remove `^`

File: app/enzyme.js (hunk 1 above)
Comment (kulczewicz) [reply app/enzyme.js:3]: Thank you for your attention, it's actually an unnecessary file, as I added later the configuration of `enzyme` is in `app/src/setupTests.js`. I also changed it to `setupTests.ts` :)

**Pull request merged**